proto/iotics/api/twin.proto

```diff
@@ -75,7 +64,7 @@ message ListAllTwinsResponse {
   // ListAllTwinsResponse twin details.
   message TwinDetails {
```

Comment:
we can return Twin - note that in protobuf all the fields are optional by definition so we can return a subset of the fileds of Twin if we choose to do so